alluxio/configuration.rst

```diff
@@ -0,0 +1,32 @@
+**********************
+Alluxio Configuration
+**********************
+
+
+|fdd| provide Alluxio with the following configurations
+
+- UnderFS: HDFS is the default |ufs|
+- Write Type: `CACHE_THROUGH`
+- Read Type: `CACHE_PROMOTE`
+
+The table below shows a list of all the available configurations.
```

Comment:
If this table shows all available configuration, also UnderFS/Write Type and Read type should be listed. However is for the other settings we keep the defaults, does if make sense to present them into our documentation?